catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPreStoragePlugin.java

```diff
@@ -471,4 +471,8 @@ public void setCreateOverview(boolean createOverview) {
     public void setStoreOriginalImage(boolean storeOriginalImage) {
         this.storeOriginalImage = storeOriginalImage;
     }
+
+    protected NitfRenderer getNitfRenderer() {
```

Comment:
❓ Since this is done only for unit testing purposes right now and we don't expect anyone to actually extend and replace the `NitfRenderer` in real scenarios, should we change this to be package-private instead of `protected`? Or, if `NitfRenderer` is actually stateless and thread-safe, we could also consider injecting it instead of creating a new one every time.